common/libs/keystore.ts

```diff
@@ -1,234 +0,0 @@
-import {
-  createCipheriv,
-  createDecipheriv,
-  pbkdf2Sync,
-  randomBytes
-} from 'crypto';
-import { privateToAddress, sha3 } from 'ethereumjs-util';
-import scrypt from 'scryptsy';
-import uuid from 'uuid';
-import { decipherBuffer, decodeCryptojsSalt, evp_kdf } from './decrypt';
-
-export interface UtcKeystore {
-  version: number;
-  id: string;
-  address: string;
-  Crypto: object;
-}
-
-// adapted from https://github.com/kvhnuke/etherwallet/blob/de536ffebb4f2d1af892a32697e89d1a0d906b01/app/scripts/myetherwallet.js#L342
-export function determineKeystoreType(file: string): string {
-  const parsed = JSON.parse(file);
-
-  if (parsed.encseed) {
-    return 'presale';
-  } else if (parsed.Crypto || parsed.crypto) {
-    return 'v2-v3-utc';
-  } else if (parsed.hash && parsed.locked === true) {
```

Comment:
The piece that identified an encrypted MEWv1 encrypted keystore file, for reference.

Reply:
@skubakdj 
scryptsy: Could you try again with a fresh `npm install`? I'm not getting any build errors.
mnemonic wallet: That would be because i forgot to wrap this class :D 
mew wallet: ill have to check this out

Reply:
MewV1 wallet should be fixed now too. Was importing from wrong module.

Reply:
The scryptsy, mnemonic, and MEWv1 encrypted wallet bugs look to be addressed :+1: 

Though I just noticed that unencrypted MEWv1 keystores will prompt for a password in the keystore decrypt component. Seems to be an issue with the `isKeystorePassRequired` function.

Reply:
This should be addressed now too

Reply:
Yep, looks good!